pelux.xml

```diff
@@ -42,16 +42,32 @@
            name="GENIVI/meta-ivi"
            path="sources/meta-ivi" />
 
-  <!-- Qt Support stuff -->
+  <!-- Qt + Qt Automotive support stuff -->
```

Comment:
meta-qt5 is not Qt Auto specific.

Reply:
I know that, but anything below that comment is only needed when we build the QtAS spin, hence the phrasing.